src/strings/strings-en.ts

```diff
@@ -622,7 +622,7 @@ export const strings = {
   PLANT: 'plant',
   PLANTING_SITE_TYPE: 'Planting Site Type',
   PLANTING_SITE_WITH_MAP_HELP:
-    'If you are a Terraformation partner or accelerator participant, please {0} to create a site with a map.',
+    'If you are a Terraformation partner or accelerator participant, please [contact us] to create a site with a map.',
```

Comment:
if we need to support valid square brackets in such a string, guessing we'll go with `\[` and `\]` if/when we need to..?

Reply:
Yeah, or we'll change the delimiter to something else. I'm not aware of any language that uses those characters in regular text, but you never know.